src/app/components/projects/pages/new/new.component.ts

```diff
@@ -62,6 +63,7 @@ ProjectNewComponent.linkToRoute({
   category: projectsCategory,
   pageRoute: newProjectMenuItem,
   menus: { actions: List(projectsMenuItemActions) },
+  renderMode: RenderMode.Client,
```

Comment:
Add a comment explaining why this route in particular is client rendering?

Reply:
Reverted